lib/ansible/modules/system/group.py

```diff
@@ -143,11 +143,18 @@ def group_mod(self, **kwargs):
         return self.execute_command(cmd)
 
     def group_exists(self):
-        try:
-            if grp.getgrnam(self.name):
-                return True
-        except KeyError:
+        if self.local:
+            with open(self.GROUPFILE) as file:
```

Comment:
use `from ansible.module_utils.facts.utils import get_file_content` for getting content of file.

Reply:
Changed the code